react-app/src/app/screens/Home/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import style from './styles.scss';
+
+class Home extends Component {
+  render() {
+    const renderLogged = logged => {
+      if (!logged) {
+        return (
+          <Fragment>
+            <span>To use the game you have to be authenticated</span>
+            <Link to="/login">Login</Link>
+          </Fragment>
+        );
+      }
+      return (
+        <Fragment>
+          <span>You are already authenticated</span>
+          <Link to="/app">Go Game</Link>
```

Comment:
I asked you to change this in the previous comment, but this comment is just for you to know. 
Notice that both cases are similar: Fragment + span + Link. You shouldn't be repeating this structure. Instead, you could define the texts and link for each case and return the same structure